contribution/zeppelinweb/goodPracticeGuide03.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: sideMenu
+title: "3 - Making A Controller"
+description: ""
+group: nav-contrib-front
+menu: nav-contrib-front
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How to Write a Controller
+
+<br/>
+Our main rule regarding writing a controller, is to use the `Controller as vm` style.
+
+However we also have a few other style guidelines.
+
+You can see below a full example of what the controller would look like.
+
```

Comment:
Do we need additional new lines? How about putting those three lines in one line? I mean

```
Our main rule regarding writing a controller, is to use the `Controller as vm` style.
However we also have a few other style guidelines. You can see below a full example of what the controller would look like.
```